molecule/broker-scale-up/converge.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Add new brokers Nodes into in-memory inventory to scale-up
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: "Add new Broker Nodes"
```

Comment:
A question for my understanding - If the host is present in the molecule file, will this module modify the host details?

Reply:
Since host is there in molecule file, create phase will bring it's container up, but It's not part of kafka_broker cluster.

In converge phase, this command will add this as a new host (duplicates are fine in ansible inventory.) But more effectively, this `add_host` module will add this node to `kafka_broker` host group for in memory inventory.
Now next platform.all playbook call will consider this change.